### PR TITLE
[FTL-1678] Move TestmailInbox into a separate package

### PR DESCRIPTION
**Pull request by ezgi-affinidi**
https://lemmatree.atlassian.net/browse/FTL-1678

### CODE REVIEW

File: sdk/core/package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "@affinidi/tools-common": "^1.0.0-beta.2",
     "@affinidi/vc-common": "^1.4.2",
     "@affinidi/vc-data": "^1.3.3",
+    "@affinidi/test-helpers": "^1.0.0",
```
Comment (inga-affinidi) [inline sdk/core/package.json:53]: Please maintain the alphabetic order here

File: sdk/test-helpers/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,667 @@
+# release 6.0.0-beta.20 (2021-10-20)
```
Comment (inga-affinidi) [inline sdk/test-helpers/CHANGELOG.md:1]: This is probably here by mistake?

File: sdk/test-helpers/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Affinity SDK for test-helpers
+
+> Please note that versions `>=4.2.6 <=5.0.0` might not work properly (see [this PR](https://github.com/affinityproject/affinidi-core-sdk/pull/105)).
```
Comment (inga-affinidi) [inline sdk/test-helpers/README.md:3]: This is probably here by mistake?

File: sdk/test-helpers/package.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  "dependencies": {
+    "node-fetch": "^2.6.0"
+  },
+  "devDependencies": {
+    "@affinidi/eslint-config": "1.0.1",
+    "@affinidi/prettier-config": "1.0.1",
+    "@types/async-retry": "^1.4.2",
+    "@types/chai": "4.2.12",
+    "@types/create-hash": "^1.2.2",
+    "@types/ethereumjs-util": "5.2.0",
```
Comment (inga-affinidi) [inline sdk/test-helpers/package.json:45]: Please remove unused dependencies

File: sdk/test-helpers/package.json [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  "typings": "dist/index.d.ts",
+  "scripts": {
+    "build": "npm run tsc",
+    "buildDocs": "typedoc --out docs src",
+    "start": "npm run build && node dist/index.js",
+    "tsc": "tsc -p tsconfig-build.json",
+    "lint": "eslint src/**",
+    "test": "",
+    "test:unit": "",
+    "test:integration": "",
```
Comment (inga-affinidi) [inline sdk/test-helpers/package.json:15]: You could have the commands there (like in core-sdk etc), just not have any actual tests, to make adding them in the future easier.

**Review by inga-affinidi — CHANGES_REQUESTED** [review]

**Comment by vitalyrudenko-affinidi** [comment]
Should we add one or two integration tests here?

**Comment by vitalyrudenko-affinidi** [comment]
Closing for the time being

**Pull request closed without merging**